### PR TITLE
add the filesystem storage driver

### PR DESCRIPTION
**Pull request by rgl**

### CODE REVIEW

File: internal/drivers/filesystem/storage.go [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+}
+
+func manifestPath(account keppel.Account, repoName, digest string) string {
+	return fmt.Sprintf("%s/%s/m/%s/%s", account.AuthTenantID, account.Name, repoName, digest)
+}
+
+// AppendToBlob implements the keppel.StorageDriver interface.
+func (d *StorageDriver) AppendToBlob(account keppel.Account, storageID string, chunkNumber uint32, chunkLength *uint64, chunk io.Reader) error {
+	path := blobPath(account, storageID)
+	tmpPath := fmt.Sprintf("%s.tmp", path)
```
Comment (SuperSandro2000) [inline internal/drivers/filesystem/storage.go:71]: Did you test what happens if two clients try to upload the same layer at the same time? My intuition tells me we should use something like ``mktemp`` here but I didn't double check if keppel prevents this already.

Comment (rgl) [reply internal/drivers/filesystem/storage.go:71]: how would this identify the client? I mean, this file must be known by the `FinalizeBlob` function bellow, so somehow that path would need to be passed around, but how?

Comment (SuperSandro2000) [reply internal/drivers/filesystem/storage.go:71]: Yeah, that is a problem. Before we merge this I need to double check if this is a real work concern or already handled by other code.

**Comment by rgl** [comment]
@SuperSandro2000, this was superseded by https://github.com/sapcc/keppel/pull/129. So I'll close this PR.

File: internal/drivers/filesystem/storage.go (hunk 1 above)
Comment (rgl) [reply internal/drivers/filesystem/storage.go:71]: @SuperSandro2000, this was superseded by https://github.com/sapcc/keppel/pull/129.

Can you please re-comment there if appropriate?

**Pull request closed without merging**